src/main/java/com/onlinebookstore/controller/BookController.java

```diff
@@ -23,36 +24,41 @@
 @Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping(value = "/api/books")
+@RequestMapping(value = "/books")
 public class BookController {
     private final BookService bookService;
 
     @Operation(summary = "Find all books", description = "Find all books")
+    @PreAuthorize("isAuthenticated()")
```

Comment:
you already have this config 
![image](https://github.com/andrey-kiyas/OnlineBookStore/assets/20326858/23774b5f-d644-4769-b836-b0b93506a17c)